src/Backend/Modules/FormBuilder/Actions/Add.php

```diff
@@ -55,6 +55,11 @@ private function loadForm()
         $this->frm->addText('email');
         $this->frm->addText('identifier', BackendFormBuilderModel::createIdentifier());
         $this->frm->addEditor('success_message');
+
+        // fields for optional confirmation mail
+        $this->frm->addCheckbox('mail_send');
```

Comment:
looking at this again, I think these variable names could be better. I'd use "send_confirmation_mail", "confirmation_mail_subject" and "confirmation_mail_content". This will make it all a little cleaner.